.github/workflows/_sitrep_mgmn.yaml

```diff
@@ -0,0 +1,116 @@
+
+on:
+  workflow_call:
+    inputs:
+      BADGE_FILENAME:
+        type: string
+        description: 'Name of the endpoint JSON file for shields.io badge'
+        required: true
+      ARTIFACT_NAME:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: true
+    outputs:
+      STATUS:
+        description: 'Summary pass/fail value indicating if results from tests are acceptable'
+        value: ${{ jobs.sitrep.outputs.STATUS }}
+
+jobs:
+  sitrep:
+    runs-on: ubuntu-22.04
+    env:
+      BADGE_FILENAME_FULL: ${{ inputs.BADGE_FILENAME }}.json
+    outputs:
+      STATUS: ${{ steps.gen-sitrep.outputs.STATUS }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Download all artifacts from the previous jobs
+        uses: actions/download-artifact@v3
+
+      - name: Write Test Status
+        shell: bash -x -e {0}
+        run: |
+          EXIT_STATUSES="${GITHUB_RUN_ID}-*/*-status.json"
+
+          cat <<EOF >>$GITHUB_STEP_SUMMARY
+          ## T5x MGMN+SPMD Test Status
+          | Test Case | State | Exit Code |
+          | --- | --- | --- |
+          EOF
+          for i in $EXIT_STATUSES; do
+            # Files are named <GHID>-<NAME>/<NAME>-status.json
+            echo "| $(echo $i | cut -d/ -f1 | cut -d- -f2) | $(jq -r .state $i) | $(jq -r .exitcode $i)"
+          done | tee -a $GITHUB_STEP_SUMMARY
+
+          echo "Test statuses:"
+          jq -rc 'input_filename,.' $EXIT_STATUSES
+
+      - name: Generate sitrep
+        id: gen-sitrep
+        shell: bash -x -e {0}
+        run: |
+          source .github/workflows/scripts/to_json.sh
+
+          EXIT_STATUSES="${GITHUB_RUN_ID}-*/*-status.json"
+          passed_tests=$(jq -r '. | select ((.state == "COMPLETED") and (.exitcode == "0")) | .state' $EXIT_STATUSES | wc -l)
+          failed_tests=$(jq -r '. | select ((.state != "COMPLETED") or (.exitcode != "0")) | .state' $EXIT_STATUSES | wc -l)
+          total_tests=$(ls $EXIT_STATUSES | wc -l)
+
+          METRICS_LOG=metrics-test-log/report.jsonl
+          all_outcomes() {
+            cat $METRICS_LOG | jq -r '. | select((.["$report_type"] == "TestReport") and (.when == "call")) | .outcome'
+          }
+          cnt_type() {
+            cat $METRICS_LOG | jq '. | select((.["$report_type"] == "TestReport") and (.when == "call") and (.outcome | contains("'${1}'"))) | .outcome' | wc -l
+          }
+          pytest_failed_tests=$(cnt_type failed)
+          pytest_passed_tests=$(cnt_type passed)
+          pytest_total_tests=$(all_outcomes | wc -l)
+
+          if ([[ $failed_tests -eq 0 ]] && [[ $total_tests -gt 0 ]] && \
+              [[ $pytest_failed_tests -eq 0 ]] && [[ $pytest_total_tests -gt 0 ]]); then
+            status=success
+            badge_color=brightgreen
+            badge_message="${passed_tests}/${total_tests} ran ${pytest_passed_tests}/${pytest_total_tests} pass loss+perf"
```

Comment:
It is a bit challenging and not quite self-explanatory as to what the differences are between the 'tests' and 'pytest tests', which I assume refer to the number of test files and tests cases, respectively. Is there a more intuitive name?

Reply:
How about:
```
"${passed_tests}/${total_tests} configs | ${pytest_passed_tests}/${pytest_total_tests} metrics"
```
?

Reply:
What do configs represent here? If it's just the jobs that ran to completion, can we use `jobs` instead of `configs` (might be easier to follow)?

Reply:
So is this the correct interpretation that a job may contain multiple tests?

Reply:
I think each job has multiple metrics to test, so yeah the tests should be a multiple of jobs I believe.

Reply:
Got it.

Reply:
Updated to 
` ${passed_tests}/${total_tests} jobs | ${pytest_passed_tests}/${pytest_total_tests} metrics`